File: Examples/CSharp/AsposePdfFacades/Stamps-Watermarks/AddImageFooter.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace CSharp.AsposePDFFacades.StampsWatermarks
{
    public class AddImageFooter
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_StampsWatermarks();

            //create PdfFileStamp object
            PdfFileStamp fileStamp = new PdfFileStamp();

            //Open Document
            fileStamp.BindPdf(dataDir + "AddImage-Footer.pdf");

            //add footer
            fileStamp.AddFooter(new FileStream(dataDir+ "aspose-logo.jpg", FileMode.Open), 10);

            //save updated PDF file
            fileStamp.Save(dataDir + "AddImage-Footer_out.pdf");

            //close fileStamp
            fileStamp.Close();
 
            
        }
    }
}

File: Examples/VisualBasic/AsposePdfFacades/Stamps-Watermarks/AddImageFooter.vb
Imports System.IO
Imports System
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades

Namespace AsposePDFFacades.StampsWatermarks
    Public Class AddImageFooter
        Public Shared Sub Run()
            ' ExStart:AddImageFooter
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_StampsWatermarks()

            ' Create PdfFileStamp object
            Dim fileStamp As New PdfFileStamp()

            ' Open Document
            fileStamp.BindPdf(dataDir & Convert.ToString("AddImage-Footer.pdf"))

            ' Add footer
            fileStamp.AddFooter(New FileStream(dataDir & Convert.ToString("aspose-logo.jpg"), FileMode.Open), 10)

            ' Save updated PDF file
            fileStamp.Save(dataDir & Convert.ToString("AddImage-Footer_out_.pdf"))

            ' Close fileStamp
            fileStamp.Close()
            ' ExEnd:AddImageFooter

        End Sub
    End Class
End Namespace
